app/models/videogame.rb

```diff
@@ -0,0 +1,5 @@
+class Videogame < ApplicationRecord
+  validates :name, presence: true
+
+  has_many :reviews
```

Comment:
We should remove line 4 since `reviews` dont exist yet.  We can add this association back in with the Reviews PR